dist/offline_installer/redhat/build_offline_installer.sh

```diff
@@ -109,5 +106,15 @@ if [ -n "$RELEASEVER" ]; then
     YUMOPTS="--releasever=$RELEASEVER"
 fi
 sudo yum -y install $YUMOPTS --downloadonly --installroot=`pwd`/build/installroot --downloaddir=build/offline_installer scylla sudo ntp ntpdate net-tools kernel-tools
+cd build
+if [ ! -x makeself-$MAKESELFVER.run ]; then
+    wget https://github.com/megastep/makeself/releases/download/release-$MAKESELFVER/makeself-$MAKESELFVER.run
+    chmod a+x makeself-$MAKESELFVER.run
+fi
+if [ ! -d ./makeself-$MAKESELFVER ]; then
+    ./makeself-$MAKESELFVER.run
+fi
```

Comment:
Why do we need this? The `makeself` version starting from Fedora 30 is 2.4.0 so at least our `dbuild` should have the correct one.

Reply:
As I mentioned on debian version https://github.com/scylladb/scylla/pull/6246#discussion_r415639584, current script only support to run in target distribution, CentOS7.
And it does not have makeself-2.4.0.